home-service/src/main/java/com/softserveinc/ita/homeproject/services/NewsService.java

```diff
@@ -0,0 +1,51 @@
+package com.softserveinc.ita.homeproject.services;
+
+import com.softserveinc.ita.homeproject.api.ApiResponseMessage;
+import com.softserveinc.ita.homeproject.api.NewsApiService;
+import com.softserveinc.ita.homeproject.api.NotFoundException;
+import com.softserveinc.ita.homeproject.model.PostNews;
+import com.softserveinc.ita.homeproject.model.UpdateNews;
+import org.springframework.context.annotation.Profile;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Service;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.SecurityContext;
+
+@Service
+@Profile("dev")
+public class NewsService implements NewsApiService {
```

Comment:
This one isn't a candidate to be a production code nor dev one. Please write tests and use it only in test scope.